src/assets/server/rest/v1/schemas/tag/tags-get.json

```diff
@@ -26,6 +26,10 @@
     },
     "ProjectFields": {
       "$ref": "common.json#/definitions/projectFields"
+    },
+    "WithUser": {
+      "type": "boolean",
+      "sanitize": "mongo"
```

Comment:
N.B: nothing to do with csv import -> allows us to see User and Email column filled with info when seeing tags list